compiler/next/include/chpl/queries/stringify-functions.h

```diff
@@ -165,20 +165,19 @@ static inline void defaultStringifyTuple(std::ostream& streamOut,
                                          const TUP& tuple,
                                          std::index_sequence<I...>) {
   // lambda to convert
-  auto convert = [](bool printsep, StringifyKind stringKind, auto& elem) {
+  auto convert = [](bool printsep, StringifyKind stringKind,
```

Comment:
might be a bit nicer if `streamOut` was in the captures list instead of as an argument

Reply:
hmm...I'm not sure how to get it out of the args because the `stringify` template needs it.